raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -465,16 +465,16 @@ contract TokenNetwork is Utils {
 
         emit NonClosingBalanceProofUpdated(channel_identifier, closing_participant, nonce);
 
-        require(channel.state == ChannelState.Closed);
+        require(channel.state == ChannelState.Closed, 'Invalid channel state.');
```

Comment:
Here too I would mention the expected channel state, e.g. 'channel state is not closed'